yellowbrick/features/decomposition.py

```diff
@@ -0,0 +1,66 @@
+##########################################################################
+## Imports
+##########################################################################
+
+from .base import FeatureVisualizer
+
+from sklearn.pipeline import Pipeline
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import StandardScaler
+
+##########################################################################
+## Decomposition Feature Visualizer
+##########################################################################
+
+class ExplainedVariance(FeatureVisualizer):
+    """
+
+    Parameters
+    ----------
+    
+
+    Examples
+    --------
+
+    >>> visualizer = ExplainedVariance()
+    >>> visualizer.fit(X)
+    >>> visualizer.transform(X)
+    >>> visualizer.poof()
+
+    Notes
+    -----
+    
+    """
+
+    def __init__(self, ax=None, n_components=None, scale=True, center=True, colormap='RdBu_r', 
```

Comment:
In the `yellowbrick.style.palettes` module we have a constant, `DEFAULT_SEQUENCE` (which is currently set to `"RdBu"`), which I'd recommend you use over a string constant for the colormap argument. Colors are so difficult, and it's also difficult to diagnose when things go wrong, so we try to handle most of our colors as variables or as simple 'b', 'g', 'r', 'm', 'y', 'c', 'k' strings which are set by the environment.